projects/storefrontstyles/scss/components/cart/_cart-coupon.scss

```diff
@@ -1,5 +1,15 @@
 @import '../../theme';
 
+%cx-cart-coupon__body {
```

Comment:
updated. not using append new class to body, instead to use angular dynamic class now

Reply:
@maggielin0 Although this is not a breaking change in itself for this scss file, we already have a **calydon** instance for this exact file. I understand that your feature may not work properly without this code so, since it's not a breaking change to the file, i'll let it slide this time, but keep in mind that all further changes **after the feature is released** will have to be done in the **calydon** instance.

Reply:
got it, thanks.